WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -880,6 +883,14 @@ private void ValidateCustomChangeAddress(IValidationErrors errors)
 
 		public override void OnOpen(CompositeDisposable disposables)
 		{
+			_isEstimateAvailabe = Observable
+				.FromEventPattern<AllFeeEstimate>(Global.FeeProviders, nameof(Global.FeeProviders.AllFeeEstimateChanged))
```

Comment:
Since we are checking for null with `Global.FeeProviders?.AllFeeEstimate`, shouldn't we do the same here?

Reply:
That doesn't work here. Also it cannot be null, but the concepts are a bit messy.